causalml/inference/meta/rlearner.py

```diff
@@ -271,6 +304,63 @@ def bootstrap(self, X, p, treatment, y, size=10000):
         te_b = self.predict(X=X)
         return te_b
 
+    def run_propensity_score(self, X, treatment, X_pred=None, treatment_pred=None, cv=None,
```

Comment:
The name of `run_propensity_score` is not straightforward to me. We don't "run" the propensity score, but either "compute", "calculate", or "predict" it.

Reply:
Yeah, it makes sense! Changed accordingly.